ricecooker/utils/youtube.py

```diff
@@ -47,7 +47,7 @@ def _get_youtube_info(self, use_proxy=True, use_cache=True, options=None):
         youtube_info = None
         # 1. Try to get from cache if allowed:
         if os.path.exists(self.cache_path) and use_cache:
-            LOGGER.info("==> [%s] Retrieving cached information...", self.__str__(), self.cachename)
+            # LOGGER.info("==> [%s] Retrieving cached information...", self.__str__(), self.cachename)
```

Comment:
Was this getting too noisy? If you switch to using `LOGGER.debug`, it should stop printing this by default.

Reply:
will do. It was giving me a logging error every time I ran the test. I'll change it to debug